docs/src/pages/[platform]/connected-components/storage/storage-browser/examples/mockConfig.ts

```diff
@@ -0,0 +1,43 @@
+import { CreateStorageBrowserInput } from '@aws-amplify/ui-react-storage/dist/types/components/StorageBrowser';
+
+export const mockConfig: CreateStorageBrowserInput['config'] = {
+  region: '',
+  registerAuthListener: () => {},
+  getLocationCredentials: () =>
+    Promise.resolve({
+      scope: '',
+      credentials: {
+        secretAccessKey: '',
+        sessionToken: '',
+        accessKeyId: '',
+        expiration: new Date(),
+      },
+    }),
+  listLocations: (input) =>
+    Promise.resolve({
+      items: [
+        {
+          bucket: 'test',
+          id: '1234',
+          permissions: ['delete', 'get', 'list', 'write'],
+          prefix: 'test',
+          type: 'PREFIX',
+        },
+        {
+          bucket: 'test-bucket',
+          id: '4567',
+          permissions: ['get', 'list'],
+          prefix: '',
+          type: 'BUCKET',
+        },
+        {
+          bucket: 'test',
+          id: '',
+          permissions: ['delete', 'get', 'list', 'write'],
+          prefix: 'some/path',
+          type: 'PREFIX',
+        },
+      ],
+      nextToken: null,
```

Comment:
Attempting to navigate into one of these locations results in an error. Is this expected?

<img width="1050" alt="Screenshot 2024-11-27 at 10 20 03 AM" src="https://github.com/user-attachments/assets/5a6e9d7d-4b8c-4969-8027-fbba447270e6">

Reply:
This works now